OracleTuxedo/core/samples/mp/Dockerfile

```diff
@@ -3,17 +3,20 @@
 # 
 
 # Pull base image
-FROM oracle/tuxedo:<VERSION>
+FROM oracle/tuxedo:12.2.2
 
-MAINTAINER Judy Liu<judy.liu@oracle.com>
-COPY simpapp_runme.sh simpappmp_runme.sh sleep.sh start_tlisten.sh /u01/oracle/
+MAINTAINER Todd Little<todd.little@oracle.com>
+COPY simpappmp_build.sh simpappmp_runme.sh sleep.sh start_tlisten.sh /u01/oracle/
```

Comment:
You should switch to the `oracle` user first (i.e. `USER oracle`) and then do the `COPY` so that you don't need to `chown` anything. You should be able to remove those lines completely.